react/features/large-video/actions.js

```diff
@@ -145,7 +145,16 @@ function _electParticipantInLargeVideo(state) {
                 //    As a last resort, pick the last participant who joined the
                 //    conference (regardless of whether they are local or
                 //    remote).
-                participant = participants[participants.length - 1];
+                //
+                // HOWEVER: We don't want to show poltergeist or other bot type participants on stage
+                // automatically, because it's misleading (users may think they are already
+                // joined and maybe speaking).
+                for (let i = participants.length; i > 0 && !participant; i--) {
```

Comment:
use array.filter?

Reply:
We could use that, but not sure if that would help, because that goes through all the elements, and now that we have this reverse order, we can shortcircuit the loop